modules/nf-core/hifiasm/environment.yml

```diff
@@ -2,4 +2,4 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - bioconda::hifiasm=0.19.8
+  - bioconda::hifiasm=0.19.9
```

Comment:
I think there's value in having version 0.19.9 in at least one version of nf-core/modules. In part because there appear to be some major changes in 0.20. We could combine the update to that version with your suggested change of the input channel.

Reply:
I'll post it as a separate issue: https://github.com/nf-core/modules/issues/6799